src/main/resources/com/google/api/codegen/php/main.snip

```diff
@@ -41,7 +43,17 @@
 @snippet generateBody(service)
     {@alwaysImport()}
 
-    // TODO generate service doc
+    /**
+    {@docLinesFromStr(serviceDocString(service))}
```

Comment:
That doesn't work actually - the stars need to be added to the beginning of each line in the service description.